OpenRA.Mods.Common/Traits/Player/StorageWarning.cs

```diff
@@ -0,0 +1,60 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Provides the player with an audible warning when their storage is nearing full.")]
+	public class StorageWarningInfo : ITraitInfo, Requires<PlayerResourcesInfo>
+	{
+		[Desc("Interval at which to check if more storage is needed.")]
+		public readonly int AdviceInterval = 250;
```

Comment:
Hmm... my goal was to preserve the existing code. Could I just change the documentation to indicate that it's in ticks?